### PR TITLE
fix crd name

### PR DESCRIPTION
**Pull request by kalantar**
Signed-off-by: Michael Kalantar <kalantar@us.ibm.com>

Fixes https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/issues/187

### CODE REVIEW

File: content/en/scenarios/deploy-kubernetes-resources.md [hunk 1]
```diff
@@ -218,4 +218,4 @@ This finalizer is supposed to be detached after the deployed local resources
 are *completely* removed from the manged cluster. With that being said, if any 
 deployed local resources are holding at the "Terminating" due to graceful 
 deletion. Both of its `ManifestWork` and `AppliedManifestWork` should stay
-undeleted.
\ No newline at end of file
+undeleted.
```
Comment (mikeshng) [inline content/en/scenarios/deploy-kubernetes-resources.md:221]: Is this change intentional?

**Comment by mikeshng** [comment]
Thank you for the PR. Looks good, just wondering about the `undeleted` change. If possible can you undo that? Maybe there is a newline special character that was introduced that is not showing on the GitHub diff?

**Comment by qiujian16** [comment]
yes, i think it is a typo on doc

/approve
/lgtm
